content/06.methods.md

```diff
@@ -125,7 +126,7 @@ This workflow is available in the [D3b GitHub repository](https://github.com/d3b
 ##### SNV and indel calling
 
 For PBTA samples, we used four variant callers to call SNVs and indels from panel, WXS, and WGS data: `Strelka2` [@doi:10.1038/s41592-018-0051-x], `Mutect2` [@biorxiv:10.1101/861054], `Lancet` [@doi:10.1038/s42003-018-0023-9], and `VarDictJava` [@doi:10.1093/nar/gkw227].
-`VarDictJava`-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives.
+`VarDictJava`-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives (~1.2M calls were included in consensus SNV calling described below).
```

Comment:
Was this number in a notebook you can point me to?

Reply:
If you read in the consensus MAF, this is the number of rows. It is also what is stated in the module README.